dcos_launch/gce.py

```diff
@@ -0,0 +1,87 @@
+import os
+import logging
+import dcos_launch.util
+from dcos_test_utils import gce
+from retrying import retry
+
+log = logging.getLogger(__name__)
+
+
+class BareClusterLauncher(dcos_launch.util.AbstractLauncher):
+    # Launches a homogeneous cluster of plain GMIs intended for onprem DC/OS
+    def __init__(self, config):
+        # config = get_hardcoded_basic_config(os_name, instance_count)
+        self.cwd = os.path.dirname(os.path.realpath(__file__))
```

Comment:
Why does this class need to know about the local path at all? In general, the launcher should know nothing about local paths. The launcher should try to keep ALL state inside the info JSON blob